constitution.md

```diff
@@ -143,26 +144,32 @@ The Outreach Committee shall be headed by the Chief Outreach Officer and shall b
 - Maintaining relationships with alums of the organization
 - Forming and maintaining professional relationships with external entities with an interest in data science, including firms, professional organizations, and academic entities at other institutions. 
 - Coordinating with the Financial Committee to obtain sponsorships from external entities.
-- Coordinating with the Programming Committee to plan and execute professional development events
+- Coordinating with the Programming Committee to plan and execute professional development events.
 
 ### Section F: Networking Committee
 The Networking Committee shall be headed by the Chief Networking Officer and shall be responsible for the following: 
 
-- Advertising GRiD membership to eligible parties on campus
-- Forming and maintaining relationships with other graduate student organizations
+- Advertising GRiD membership to eligible parties on campus.
+- Forming and maintaining relationships with other graduate student organizations.
 - Forming and maintaining relationships with faculty members whose teaching and/or research interests involve data science and statistics.
-- Coordinating with the Programming Committee to plan and execute networking events
-- Maintaining a list of data science other events on campus and in the region that may be of interest to GRiD members
+- Coordinating with the Programming Committee to plan and execute networking events.
+- Maintaining a list of other data science events on campus and in the region that may be of interest to GRiD members.
 
 ## ARTICLE V: ELECTIONS AND VOTING
 <a name="elections"></a>
 ### Section A: Election 
 
-The Nomination Period for organization elections will commence at the final meeting of the fall semester and conclude at the first meeting of the spring semester. Nominations may be solicited and accepted at either meeting or in the form of email to the board during the winter break. Any member may nominate a member (including him/herself) for any Executive Board Officer position.
+The Nomination Period for organization elections will commence at the final meeting of the fall semester and conclude at the first meeting of the spring semester. Nominations may be solicited and accepted at either meeting or in the form of email to the board during the winter break. Any member may nominate any other member (including herself/him) for any Executive Board Officer position.
```

Comment:
"him/herself" or "her/himself" is more accepted. Although I have become a proponent of the singular they in cases like this. I guess that would be "themself" http://blog.oxforddictionaries.com/2013/01/themself/.